modules/logging-loki-retention.adoc

```diff
@@ -10,17 +10,15 @@ With Logging version 5.6 and higher, you can configure retention policies based
 
 . To enable stream-based retention, create a `LokiStack` custom resource (CR):
 +
-.Example global stream-based retention
+.Example global stream-based retention for AWS
 [source,yaml]
 ----
 apiVersion: loki.grafana.com/v1
 kind: LokiStack
 metadata:
   name: logging-loki
   namespace: openshift-logging
-spec:
-  limits:
-    global: <1>
+   global: <1>
```

Comment:
It is better to use the default class name is gp2 in AWS.   storageClassName: gp2

Reply:
retention is cloud irrelevant. I think one example is enough.

Reply:
I have removed the GCP examples and changed storageClassName for AWS examples.